packages/client/src/__tests__/integration/errors/source-map-support/__snapshots__/test.ts.snap

```diff
@@ -6,12 +6,9 @@ Invalid \`prisma.user.findUnique()\` invocation:
 
 {
   where: {
-    id: null
-        ~~~~
++   id: String
```

Comment:
This error looks to me as if `id` wasn't specified at all and not explicitly set to `null`, probably something to look into